### PR TITLE
edit cmd/genswaggertypedocs.go log to structured Log

### PR DESCRIPTION
**Pull request by LinshanYu**
/kind documentation

edit cmd/genswaggertypedocs.go log to structured Log，https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

### CODE REVIEW

File: cmd/cloud-controller-manager/main.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func main() {
 
 	s, err := options.NewCloudControllerManagerOptions()
 	if err != nil {
-		klog.Fatalf("unable to initialize command options: %v", err)
+		klog.ErrorS("unable to initialize command options: %v", err)
```
Comment (cheftako) [inline cmd/cloud-controller-manager/main.go:68]: Please review https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/. Also err should be the first parameter to ErrorS

**Comment by stewart-yu** [comment]
any feedback to comments in this PR? @LinshanYu

**Comment by yangjunmyfm192085** [comment]
@LinshanYu need rebase. and  use the right format

**Pull request closed without merging**